modules/ROOT/assets/attachments/sg.yaml

```
@@ -848,3 +960,13 @@ properties:
   SSLKey:
     description: Absolute or relative path on the filesystem to the TLS private key file, if TLS is used to secure Sync Gateway connections, or "nil" for plaintext. A relative path is from the directory that contains the Sync Gateway executable file.
     type: string
+  tls_minimum_version:
+    description: |
+      Enforce a minimum TLS version to be used in replications with Couchbase Lite.
+
+      Possible values are:
+
+      - "tlsv1"
```

Sorry, I typo'd possible options in the Jira. Need `tlsv1.1` in there too.

```suggestion
      - "tlsv1"
      - "tlsv1.1"
```